language/stdlib/modules/LibraSystem.move

```diff
@@ -374,10 +372,13 @@ module LibraSystem {
     // This function should never throw an assertion.
     fun update_ith_validator_info_(validators: &mut vector<ValidatorInfo>, i: u64): bool {
         let size = Vector::length(validators);
+        // TODO (dd): This provably cannot happen
```

Comment:
It is an interesting question whether we should remove the code in such cases. I would say no, for robustness. If you agree, I would remove the TODO but instead just add a comment like "Provably does not happen, but here for sanity".